api_schemas/event_schemas.py

```diff
@@ -29,3 +29,10 @@ class EventCreate(BaseSchema):
     title_en: Annotated[str, StringConstraints(max_length=MAX_EVENT_TITLE)]
     description_sv: Annotated[str, StringConstraints(max_length=MAX_EVENT_DESC)]
     description_en: Annotated[str, StringConstraints(max_length=MAX_EVENT_DESC)]
+
+
+class EventUpdate(BaseSchema):
+    title_sv:Annotated[str, StringConstraints(max_length=MAX_EVENT_TITLE)]
+    title_en:Annotated[str, StringConstraints(max_length=MAX_EVENT_TITLE)]
+    description_sv: Annotated[str, StringConstraints(max_length=MAX_EVENT_DESC)]

```

Comment:
To make a proper PATCH route, all fields should be optional. An optional string is defined on a Pydantic schema as `str | None` (works the same event if wrapped in `Annotated`)

Reply:
jag har en fixat en nu men problemet är att den kommer inte bli None för något värde på de olika stringsen pga hur man skickar in i JSON tror jag! :/